rpcs3/Emu/Cell/SPUThread.cpp

```diff
@@ -2773,6 +2774,8 @@ bool spu_thread::stop_and_signal(u32 code)
 
 	if (offset >= RAW_SPU_BASE_ADDR)
 	{
+		// Save next PC and current SPU Interrupt Status
+		npc = (pc + 4) | (interrupts_enabled);
```

Comment:
its already done in spu_thread::cpu_stop() ?

Reply:
Fixed I hope